constants/megaMenuLinks.ts

```diff
@@ -611,6 +639,27 @@ const QUANTUM_ALGORITHMS_FOR_APPS : MegaDropdownMenuGroup = {
       segment: {
         cta: 'flexible-representation-of-quantum-images-frqi', location: sectionApps
       }
+    },
+    {
+      label: 'Quantum Facial Expression Recognition',
+      url: `${baseUrl}${pathApps}/facial-expression-recognition`,
+      segment: {
+        cta: 'facial-expression-recognition', location: sectionApps
+      }
+    },
+    {
+      label: 'Quantum edge detection',
```

Comment:
This could use the same casing for consistency `Quantum Edge Detection`

Reply:
I have kept it as 'Quantum edge detection' because the actual page has this title.
https://learn.qiskit.org/course/ch-applications/quantum-edge-detection